monai/networks/nets/hovernet.py

```diff
@@ -27,19 +27,26 @@
 # }
 # =========================================================================
 
+import os
+import re
 from collections import OrderedDict
 from typing import Callable, Dict, List, Optional, Sequence, Type, Union
 
 import torch
 import torch.nn as nn
 
+from monai.apps.utils import download_url
 from monai.networks.blocks import UpSample
 from monai.networks.layers.factories import Conv, Dropout
 from monai.networks.layers.utils import get_act_layer, get_norm_layer
 from monai.utils.enums import HoVerNetBranch, HoVerNetMode, InterpolateMode, UpsampleMode
 from monai.utils.module import export, look_up_option
 
-__all__ = ["HoVerNet", "Hovernet", "HoVernet", "HoVerNet"]
+__all__ = ["HoVerNet", "Hovernet", "HoVernet", "HoVerNet", "HOVERNET_MODELS"]
+
+HOVERNET_MODELS = {
+    "preact-resnet50": "https://drive.google.com/u/1/uc?id=1KntZge40tAHgyXmHYVqZZ5d2p_4Qr2l5&export=download"
```

Comment:
could you move this to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 which is more stable. and if this is trained from imagenet we may need comment on the license.

Reply:
Thanks @wyli , this link comes from the referred HoverNet repo: https://github.com/vqdang/hover_net#data-format
the author uploaded the weights into the google drive and I just copy the google drive link. Could we download and copy to releases? Or let me just add some doc strings?

Reply:
Hi @yiheng-wang-nv, I don't think that we can add their weight to our release since it is trained on a data with "CC BY-NC" license, which is more restrictive than MONAI license (Apache 2.0). I suggest to check with @daguangxu on that.

Reply:
@yiheng-wang-nv, please remove the link to gdrive from here and just keep the comment about the existence of pretrained weights with the link to the website. You can add the link to gdrive in the tutorial with appropriate comments about the license as we discussed.